build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -2,6 +2,21 @@
 apr-util-ldap:
```

Comment:
clamav* modules have all been updated.   New version clamav-0.100.1-3.el7.x86_64.rpm   I would have updated them but I am not sure about libNX stuff at this point.  (See later comment)